### PR TITLE
Add support for ed25519 ssh key encryption

### PR DESCRIPTION
**Pull request by rafzei**
I also had an issue with running spec tests with 3072 bit RSA key

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -41,7 +41,7 @@ RUN : INSTALL HELM BINARY \
 
 RUN : INSTALL GEM REQUIREMENTS \
  && gem install \
-    rake rspec_junit_formatter serverspec solargraph rubocop rubocop-junit_formatter
+    rake rspec_junit_formatter serverspec solargraph rubocop rubocop-junit_formatter bcrypt_pbkdf ed25519
```
Comment (to-bar) [inline .devcontainer/Dockerfile]: How about sorting these gems? :)

**Review by przemyslavic — DISMISSED** [review]

**Review by przemyslavic — COMMENTED** [review]
Consider adding it also to the `epicli` container as spec tests are part of it.
https://github.com/epiphany-platform/epiphany/blob/develop/Dockerfile

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (rafzei) [reply .devcontainer/Dockerfile]: Alphabetical order especially for you!

**Comment by przemyslavic** [comment]
Experiencing issues with building an `epicli` image. Reproduced locally and in the pipeline.
```
ERROR:  Error installing bcrypt_pbkdf:
	ERROR: Failed to build gem native extension.

    current directory: /var/lib/gems/2.7.0/gems/bcrypt_pbkdf-1.1.0/ext/mri
/usr/bin/ruby2.7 -I /usr/lib/ruby/vendor_ruby -r ./siteconf20211209-1156-okxfa7.rb extconf.rb
creating Makefile

current directory: /var/lib/gems/2.7.0/gems/bcrypt_pbkdf-1.1.0/ext/mri
make DESTDIR\= clean

current directory: /var/lib/gems/2.7.0/gems/bcrypt_pbkdf-1.1.0/ext/mri
make DESTDIR\=
compiling bcrypt_pbkdf.c
bcrypt_pbkdf.c:18:10: fatal error: sys/types.h: No such file or directory
   18 | #include <sys/types.h>
      |          ^~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:245: bcrypt_pbkdf.o] Error 1

make failed, exit code 2

Gem files will remain installed in /var/lib/gems/2.7.0/gems/bcrypt_pbkdf-1.1.0 for inspection.
Results logged to /var/lib/gems/2.7.0/extensions/x86_64-linux/2.7.0/bcrypt_pbkdf-1.1.0/gem_make.out
Building native extensions. This could take a while...

ERROR:  Error installing ed25519:
	ERROR: Failed to build gem native extension.

    current directory: /var/lib/gems/2.7.0/gems/ed25519-1.2.4/ext/ed25519_ref10
/usr/bin/ruby2.7 -I /usr/lib/ruby/vendor_ruby -r ./siteconf20211209-1156-js1gdp.rb extconf.rb
creating Makefile

current directory: /var/lib/gems/2.7.0/gems/ed25519-1.2.4/ext/ed25519_ref10
make DESTDIR\= clean

current directory: /var/lib/gems/2.7.0/gems/ed25519-1.2.4/ext/ed25519_ref10
make DESTDIR\=
compiling ed25519_ref10.c
In file included from /usr/include/ruby-2.7.0/ruby/ruby.h:29,
                 from /usr/include/ruby-2.7.0/ruby.h:33,
                 from ed25519_ref10.c:1:
/usr/include/ruby-2.7.0/ruby/defines.h:126:10: fatal error: stdio.h: No such file or directory
  126 | #include <stdio.h>
      |          ^~~~~~~~~
compilation terminated.
make: *** [Makefile:245: ed25519_ref10.o] Error 1

make failed, exit code 2

Gem files will remain installed in /var/lib/gems/2.7.0/gems/ed25519-1.2.4 for inspection.
Results logged to /var/lib/gems/2.7.0/extensions/x86_64-linux/2.7.0/ed25519-1.2.4/gem_make.out

```

**Comment by przemyslavic** [comment]
Looks like we have to install `libc6-dev` as well. 
https://stackoverflow.com/questions/19580758/gcc-fatal-error-stdio-h-no-such-file-or-directory/20150282

edit:
I can confirm it has helped.
Please add it here: https://github.com/epiphany-platform/epiphany/blob/47dafff8631c7040ba1ee7011f3e03a3bbd6f292/Dockerfile#L18

**Review by to-bar — APPROVED** [review]

**Review by seriva — APPROVED** [review]

**Review by przemyslavic — APPROVED** [review]

**Pull request merged**